src/api.ts

```diff
@@ -47,4 +48,21 @@ router.get('/moderationList', async (req, res) => {
   }
 });
 
+router.post('/nftClaimer/:type(space|proposal)/sign', async (req, res) => {
```

Comment:
Why route path is in cameCase? Do we explicitly enabled it in express?
Links:
https://www.synopsys.com/blogs/software-security/preventing-common-expressjs-mean-stack-vulnerabilities
https://stackoverflow.com/questions/21216523/nodejs-express-case-sensitive-urls
https://ux.stackexchange.com/questions/41595/what-is-the-casing-convention-for-url-routes

Reply:
> Do you want to go ahead and review+merge this PR, so you can use this instead of mocking the trusted backend, or do you want to wait for everyone to go live first ?

I think it would be great to wait for live version, if it wont take too long.

Reply:
otherwise looks good